src/runtime.js

```diff
@@ -119,6 +120,7 @@ export function installRuntimeServices(global) {
   installTimerService(global);
   installVsyncService(global);
   installXhrService(global);
+  installCachedXhrService(global);
```

Comment:
Ah that makes sense, I had just run `build`. What do you think of adding `dist` as a gulp task dependency for the `size` task? Are there any cases where we want to run `size` on files produced by a task other than `dist`?